compute/etc/sql_exporter/compute_max_connections.sql

```diff
@@ -0,0 +1 @@
+SELECT setting AS max_connections FROM pg_catalog.pg_settings WHERE name='max_connections';
```

Comment:
I think you can do something like `SELECT current_setting('max_connections')`

Reply:
Fixed